Documentation/Upgrade/rook-upgrade.md

```diff
@@ -129,9 +115,9 @@ In order to successfully upgrade a Rook cluster, the following prerequisites mus
 
 ## Rook Operator Upgrade
 
-In the examples given in this guide, we will be upgrading a live Rook cluster running `v1.8.9` to
-the version `v1.9.3`. This upgrade should work from any official patch release of Rook v1.8 to any
-official patch release of v1.9.
+In the examples given in this guide, we will be upgrading a live Rook cluster running `v1.9.10` to
```

Comment:
Of note, I assume we will release 1.9.10 before 1.10.0.

Reply:
Sounds good, I'm thinking we'll release them the same day.